testfiles/1ZoneDataCenterCRAC_wApproachTemp.idf

```diff
@@ -796,7 +796,7 @@
 
   ElectricEquipment:ITE:AirCooled,
     Data Center Servers,     !- Name
-    Main Zone,               !- Zone Name
+    Main Zone,               !- Zone or Space Name
```

Comment:
This field name change needs to be described in Rules22-1-0-to-22-2-0.md

Reply:
> This field name change needs to be described in Rules22-1-0-to-22-2-0.md

ElectricEquipment:ITE:AirCooled Zone or Space Name was introduced in a prior release, but apparently the example files were not updated.